en/docs/develop/dotnet-sdk.md

```diff
@@ -0,0 +1,98 @@
+# Using the .NET SDK
+
+This page guides you through implementing authentication for a .NET client application using the OpenID Connect authentication APIs with WSO2 Identity Server.
+
+----
+
+## Get started
+
+To use the SDK in your client application project, you can either build from source or use the Nuget package manager. 
+
+**Build from source**
+
+1. Clone the .NET sdk git repository using the following command. 
+    ```
+    git clone https://github.com/wso2-extensions/identity-sdks-dotnet.git
+    ```
+
+2. Open the solution using Visual Studio.
+
+3. Build the project. 
+
+4. Add the `org.wso2.identity.sdk.oidc.dll` file to your client application project. 
+
+**Use Nuget package manager**
+
+1. Open the Nuget package manger.
+
+2. Search for `org.wso2.identity.sdk.oidc` or navigate to [this link](https://www.nuget.org/packages/org.wso2.identity.sdk.oidc/).
+
+3. Include it with the suggested required dependencies. 
+
+Alternatively, you can also run the following command in the package manager CLI as shown below. 
+
+```
+Install-Package org.wso2.identity.sdk.oidc -Version 1.0.0
+```
+
+----
+
+## Using the SDK
+
+Below explained key API methods that is available in the OIDC SDK and how to use them.
```

Comment:
Shall we rephrase this?
The key API methods that are available in the OIDC SDK and the way to configure them are explained below.

Reply:
Actually meant to remove this sentence as it not really neccessary. Removed it now.